spec/models/fax_recipient_spec.rb

```diff
@@ -1,135 +1,77 @@
-          residential_address = double(zip: "12345")
-          office_location = "union"
-          snap_application = double(
-            residential_address: residential_address,
-            office_location: office_location,
-          )
-
-          fax_recipient = described_class.new(
-            snap_application: snap_application,
-          )
-
-          expect(fax_recipient.number).to eq union_fax_number
-        end
-      end
-
-      it "uses the appropriate fax number when the address is in clio" do
-        residential_address = double(zip: "48415")
-        snap_application = double(
-          residential_address: residential_address,
-          office_location: nil,
-        )
-
-        fax_recipient = described_class.new(
-          snap_application: snap_application,
-        )
-
-        expect(fax_recipient.number).to eq clio_fax_number
-      end
+    it "falls back to the clio office" do
+      fax_recipient = described_class.new(
+        snap_application: app(from_zip_covered_by: :nobody),
+      )
 
-      it "uses the appropriate fax number when the address is in union" do
-        residential_address = double(zip: "48411")
-        snap_application = double(
-          residential_address: residential_address,
-          office_location: nil,
-        )
+      expect(fax_recipient.office).to eq("name" => "Clio",
+                                         "fax_number" => clio_fax_number)
+    end
 
-        fax_recipient = described_class.new(
-          snap_application: snap_application,
-        )
+    it "allows for testing via zip 12345" do
+      fax_recipient = described_class.new(
+        snap_application: app(from_zip_covered_by: :testing),
+      )
 
-        expect(fax_recipient.number).to eq union_fax_number
-      end
+      expect(fax_recipient.office).to eq("name" => "Staging Sfax",
+                                         "fax_number" => staging_fax_number)
+    end
+  end
 
-      it "falls back to the clio office" do
-        residential_address = double(zip: "123456")
+  describe "#number" do
+    it "is whatever the office number is" do
+      fax_recipient = described_class.new(
+        snap_application: app(from_zip_covered_by: :testing),
+      )
 
-        snap_application = double(
-          residential_address: residential_address,
-          office_location: nil,
-        )
+      expect(fax_recipient.number).to eq fax_recipient.office["fax_number"]
+    end
+  end
 
-        fax_recipient = described_class.new(
-          snap_application: snap_application,
-        )
+  describe "#name" do
+    it "is whatever the office name is" do
+      fax_recipient = described_class.new(
+        snap_application: app(from_zip_covered_by: :testing),
+      )
 
-        expect(fax_recipient.number).to eq clio_fax_number
-      end
+      expect(fax_recipient.name).to eq fax_recipient.office["name"]
     end
   end
-
   def sandbox_fax_number
```

Comment:
looks like we removed tests for sandbox fax number but did not re-add any?

Reply:
Yeah, I removed them because they weren't as important as the non-production ones, and forgot to clean out this method.